providers/loopia/loopiaProvider.go

```diff
@@ -0,0 +1,426 @@
+package loopia
+
+/*
+
+Loopia XML_RPC API V1 DNS provider:
+
+Documentation: https://www.loopia.com/api/
+Endpoint: https://api.loopia.se/RPCSERV
+
+Settings from `creds.json`:
+   - username
+   - password
+   - customer_number (for resellers)
+
+*/
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff2"
+	"github.com/StackExchange/dnscontrol/v3/pkg/printer"
+	"github.com/StackExchange/dnscontrol/v3/pkg/txtutil"
+	"github.com/StackExchange/dnscontrol/v3/providers"
+	"github.com/miekg/dns/dnsutil"
+)
+
+// Section 1: Register this provider in the system.
+
+// init registers the provider to dnscontrol.
+func init() {
+	fns := providers.DspFuncs{
+		Initializer:   newDsp,
+		RecordAuditor: AuditRecords,
+	}
+	providers.RegisterDomainServiceProviderType("LOOPIA_XMLRPCv1", fns, features)
+	providers.RegisterRegistrarType("LOOPIA_XMLRPCv1", newReg)
```

Comment:
Can we call this something more simple than "LOOPIA_XMLRPCv1"?  How about just "LOOPIA"?  That way it matches the directory name and the fooProvider.go name.

Reply:
Yeah, I had a similar thought, but I had the hope that one day they might release a better API. I suppose at that point it's possible rename the existing stuff.